gapis/api/service.proto

```diff
@@ -145,6 +145,7 @@ message Shader {
   string source_language = 3;
   string spirv_source = 4;
   bool cross_compiled = 5;
+  DataGroup static_analysis = 6;
```

Comment:
I think we've reached the point, where I'd prefer we'd define an explicit proto for this, rather than stick with the server generated key-value map approach. In this PR you're adding code to format all kinds of values, even though all we're currently (and probably ever) are going to get here are uint32s.

Reply:
I went ahead and updated it to a more explicit implementation. However, I am starting to think that the better approach would be to have an array of ints and an array of corresponding labels for said ints. Thoughts?